http/spec.go

```diff
@@ -121,8 +121,8 @@ func (s *Service) Spec(ctx context.Context,
 
 		// Handle durations.
 		if strings.HasPrefix(k, "SECONDS_PER_") || k == "GENESIS_DELAY" {
-			intVal, err := strconv.ParseUint(v, 10, 64)
-			if err == nil && intVal != 0 {
+			intVal, err := strconv.ParseInt(v, 10, 64)
+			if err == nil && intVal >= 0 {
```

Comment:
This allows a zero value now. Is that intended?

Reply:
Same as above. coming from master (https://github.com/attestantio/go-eth2-client/commit/50179b8bd4d75c956248c8af698373fac1ddb6c9).